0031-windows-install-anywhere.md

```diff
@@ -0,0 +1,170 @@
+- Feature Name: Windows Install Anywhere
+- Start Date: 2021-01-19
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Allow the user to select the install location for salt in the Windows Installer.
+Default to Program Files
+
+# Motivation
+[motivation]: #motivation
+
+Windows is not Linux
+
+The Windows directory structure is such that installed software should reside in
+the ``Program Files`` directory on the ``C:`` drive. Software should not be
+installed on the root of the system directory. Salt is currently installed on
+the root of the ``C:`` drive. This seems to be a paradigm brought over from
```

Comment:
> This seems to be a paradigm brought over from Linux.

What does this mean. I have never, ever seen anything installed directly at the root on Linux. The closest you get is people dumping stuff under `/opt` and not using `/etc/opt` or `/var/opt`.

Reply:
Perhaps this was overstated and not well written out. What we wanted to point out from this specific paragraph was just simply that we have the following:
```
C:\salt
C:\salt\var
C:\salt\bin
C:\salt\srv
```
and this was something that was carried over from linux whereas this convention is just simply not correct for Windows. Therefore, we wish to remove prior linux conventions.